pkgs/by-name/pa/pandoc-mustache/package.nix

```
@@ -0,0 +1,37 @@
+{
+  python3Packages,
+  fetchPypi,
+  lib,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "pandoc-mustache";
+  version = "0.1.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-xGjRwhZ2zx+YJARaB4/l1OhFauy/LMosIjMQqoGnxrM=";
+  };
+
+  build-system = with python3Packages; [
+    setuptools
+    pyparsing
+  ];
+
+  dependencies = with python3Packages; [
+    panflute
+    pystache
+    pyyaml
+    future
+  ];
+
+  meta = {
+    description = "Pandoc Mustache Filter";
+    homepage = "https://github.com/michaelstepner/pandoc-mustache";
```

Consider adding changelog
```suggestion
    homepage = "https://github.com/michaelstepner/pandoc-mustache";
    changelog = "https://github.com/michaelstepner/pandoc-mustache/releases/tag/${version}/CHANGELOG.md"
```